pkg/gitconfig/gitconfig_test.go

```diff
@@ -343,7 +342,8 @@ func TestParseComplex(t *testing.T) {
 
 	c := ParseConfig(strings.NewReader(configSampleComplex))
 
-	assert.Contains(t, maps.Keys(c.vars), "core.sshCommand")
+	_, ok := c.vars["core.sshCommand"]
+	assert.True(t, ok)
```

Comment:
why not use `SortedKeys` here?

Reply:
The old code

```go
assert.Contains(t, maps.Keys(c.vars), "core.sshCommand")
```

was just testing whether the `core.sshCommand` key exists in the `c.vars` map or not.

Obviously we can do `assert.Contains(t, set.SortedKeys(c.vars), "core.sshCommand")` too to test whether `core.sshCommand` exists in `c.vars` or not. But `_, ok := c.vars["core.sshCommand"]` does the job in a much simplier way